Branch-SDK/Branch-SDK/Branch.h

```diff
@@ -154,6 +155,33 @@ typedef NS_ENUM(NSUInteger, BranchCreditHistoryOrder) {
  */
 + (Branch *)getInstance:(NSString *)branchKey;
 
+/// TODO: Document:
```

Comment:
Yeah, unfortunately class properties don't exist prior to Xcode 8.

I've defined these class methods in a 'class property' style, so maybe we can turn this on later.